snuba/manual_jobs/runner.py

```diff
@@ -0,0 +1,40 @@
+import os
+from typing import Any, Mapping
+
+import simplejson
+
+from snuba.manual_jobs import JobSpec
+
+
+def _read_from_path(filename: str) -> Mapping[str, JobSpec]:
+    local_root = os.path.dirname(__file__)
+
+    with open(os.path.join(local_root, filename)) as stream:
+        contents = simplejson.loads(stream.read())
+
+        job_specs = {}
+        for content in contents:
+            job_spec = _job_spec_from_entry(content)
+            job_specs[job_spec.job_id] = job_spec
+        return job_specs
+
+
+def _job_spec_from_entry(content: Any) -> JobSpec:
```

Comment:
can we do _get_job_spec

Reply:
how about `_extract` or `_build`? get specifically just seems kind of vague to me

Reply:
youre right, build is bettere than get